src/context/src/rendering_context.rs

```diff
@@ -0,0 +1,90 @@
+extern crate amethyst_renderer;
+extern crate glutin;
+extern crate gfx_window_glutin;
+extern crate gfx_device_gl;
+extern crate gfx;
+
+use amethyst_config::Element;
+use std::path::Path;
+use self::glutin::{Window, WindowBuilder};
+use self::amethyst_renderer::Renderer;
+use self::amethyst_renderer::target::{ColorFormat, DepthFormat};
+use self::gfx_device_gl::{Device, Factory, Resources, CommandBuffer};
+use self::gfx::handle::{RenderTargetView, DepthStencilView};
+
+config!(
+    /// Contains display config,
+    /// it is required to create a `(Window, RenderingContext)` pair
+    struct DisplayConfig {
+        pub title: String = "Amethyst game".to_string(),
+        pub fullscreen: bool = false,
+        pub dimensions: Option<(u32, u32)> = None,
```

Comment:
May want to change this to just `(u32, u32)` instead of an `Option`, since otherwise it would be defaulted to `None` and the window won't know what the default resolution is.